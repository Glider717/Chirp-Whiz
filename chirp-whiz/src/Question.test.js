import React from 'react';

const QuestionInfo = require('./Question.js');

test('QuestionInfo returns correct', () => {
    expect(QuestionInfo({type: 0, bird: {name: "a", image: "b", sound: "c"}})).toStrictEqual(<h3>a</h3>);
});