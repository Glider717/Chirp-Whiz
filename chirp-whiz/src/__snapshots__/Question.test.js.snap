// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Question renders 1`] = `undefined`;
